xcm/xcm-builder/src/origin_conversion.rs

```diff
@@ -253,8 +253,8 @@ where
 	}
 
 	#[cfg(feature = "runtime-benchmarks")]
-	fn successful_origin() -> Origin {
-		Origin::root()
+	fn try_successful_origin() -> Result<Origin, ()> {
```

Comment:
Was this renamed in substrate?

Reply:
https://github.com/paritytech/substrate/pull/11558